crowbar_framework/config/locales/suse_manager_client/en.yml

```diff
@@ -22,3 +22,11 @@ en:
         manager_server_hint: 'For example susemgr.example.com. You can set a record within the %{link}.'
         activation_key: 'Activation Key'
         activation_key_hint: 'For example 1-00112233445566778899aabbccddeeff'
+
+        manual_steps: |
+          In order to use this barclamp, you have to execute some manual
+          steps. After installing the SUSE Manager Server you have to
+          download the RPM from <strong>%{manager_url}</strong> and place
+          it at <strong>%{rpm_path}</strong>. To finish the preparation you
+          have to execute <strong>%{install_command}</strong> on the
+          Administration Server.
```

Comment:
Bah, not sure how I missed them, but I'd still add some commas and rephrase further. I'd also remove the "you", so let me rephrase this to:
```
Manual steps are required in order to use this barclamp. After
installing the SUSE Manager Server, download
<strong>%{manager_url}</strong> and place the file at
<strong>%{rpm_path}</strong>. To finish the setup, execute
<strong>%{install_command}</strong> on the Administration
Server.
```